### ISSUE TITLE
`fps_only` config resizes the label when FPS is 100 or 10

### ISSUE THREAD

**Issue by zaps166** [flightlessmango/MangoHud#1342]
When `MANGOHUD_CONFIG` is set to `fps_only`, the label resizes non-stop when FPS is 100 or 10 causing FPS counter to change position os screen when it's aligned to the right.

MangoHud version: 0.7.2.

**Steps to reproduce the behavior**
1. Execute: `MANGOHUD=1 MANGOHUD_CONFIG=fps_only,position=top-right,fps_limit=10 vkcube`.
2. Observe FPS counter.

**Expected behavior**
Don't resize the label as long as FPS counter shows constant value.

**Screenshots**

https://github.com/flightlessmango/MangoHud/assets/6239090/7b35ae8a-718f-4aa0-9fe7-b947dfe2ff00

**Comment by gort818** [comment]
Does this happen if you set `position=top-left` ?

**Comment by zaps166** [comment]
> Does this happen if you set `position=top-left` ?

Yes, I see the semi-transparent background of FPS label is resizing, too.

---

Looks like size is determined on different number rounding maybe (example: 9.99 VS 10.00)?
